satellite/satellitedb/projectmembers.go

```diff
@@ -71,20 +71,17 @@ func (pm *projectMembers) GetByProjectID(ctx context.Context, projectID uuid.UUI
 
 		scanErr := rows.Scan(&memberIDBytes, &projectIDBytes, &pm.CreatedAt)
 		if err != nil {
-			err = errs.Combine(err, scanErr)
-			continue
+			return nil, scanErr
```

Comment:
The line above should be `if scanErr != nil`.

Perhaps, it would be less error-prone to reuse `err` in all three places instead of `scanErr` and `convertErr`.